source/schwarz_base.cpp

```diff
@@ -136,14 +139,18 @@ void SchwarzBase<ValueType, IndexType>::initialize(
     // if explicit_laplacian has been enabled or an external matrix has been
     // provided.
     if (settings.explicit_laplacian || settings.matrix_filename != "null") {
+#if !SCHW_HAVE_DEALII
```

Comment:
I think here you might run into problems as the new one works properly with deal.ii and this might not as it is the older interface.